### ISSUE TITLE
Unable to generate http response

### ISSUE THREAD

**Issue by RaulTrombin** [paperclip-rs/paperclip#513]
```
#[api_v2_operation]
#[post("/echo")]
async fn echo(req_body: String) -> Result<HttpResponse, Error> {
    Ok(HttpResponse::Ok().body(req_body))
}
```

Isn't generating the Ok response to apiv2

Structural error at paths./echo.post.responses
should define at least one response

in the examples we use Result<Json<structure>, Error>, 
Can someone help on how to do this for other types?

**Comment by tiagolobocastro** [comment]
We cannot the information from dynamic objects such as the `HttpResponse`. 
The type you mentioned `Json` works because it allows you to specify which type you want to set as the return, example:
```rust

/// Here we can generate all the information we need at compile time
#[derive(Serialize, Deserialize, Apiv2Schema)]
struct Pet { ... }

/// Here we set the return type of Json<Pet> and we internally generate 
/// all the information for Json<T> where T: Apiv2Schema
-> Result<Json<Pet>, Error>
```

IIRC You can also return an object of your own type wrapper (eg: MyWrapper<T: Apiv2Schema>)

**Comment by RaulTrombin** [comment]
```
#[api_v2_operation]
#[get("/")]
async fn index() -> impl Responder {
    handle_embedded_file("index.html")
}
#[api_v2_operation]
#[get("/dist/{_:.*}")]
async fn dist(path: web::Path<String>) -> impl Responder {
    handle_embedded_file(path.as_str())
}
#[api_v2_operation]
#[post("/echo")]
async fn echo(req_body: String) -> HttpResponse {
    // body(req_body).
    // Ok(HttpResponse::Ok().finish())
    (HttpResponse::build(StatusCode::OK).body(req_body))
}
```

for echo it's fine to have a type as answer, but if it's just an index or websocket's route, how can we do it?
I don't get why the paperclip can't take the "status:Okay" from HttpResponse::Ok().

**Comment by tiagolobocastro** [comment]
I agree that it would be good but how would paperclip know that? The spec is generated at compile time so we'd need to parse all your code and try to infer the return type from it.
Sadly at the moment there's no great alternative to using a type or manually adding details in the macro.
If you want a type that just sets 204, then you can always add your own type wrapper of HttpResponse, implement Responder etc

**Comment by RaulTrombin** [comment]
Somehow I was unable to show responses on the openapi specs page.
Then I found the skip feature for api_v2:
#[api_v2_operation(skip)]

Should be fine for index at "/"
I was trying to make a valid openapi spec and finally get it skipping "/" and "/websocket" .
![image](https://github.com/paperclip-rs/paperclip/assets/80598030/3d5dfa58-6d25-43da-a77a-f1bc46bbe99b)
